### PR TITLE
Improve docs for std.range.dropExactly

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

**Comment by atilaneves** [comment]
LGTM

File: std/range/package.d [hunk 1]
```diff
@@ -2833,6 +2833,13 @@ R dropBack(R)(R range, size_t n)
     on an empty range, which is undefined behavior. So, only use
     $(D popFrontExactly) when it is guaranteed that $(D range) holds at least
     $(D n) elements.
+
+    Params
+        r = the input range to drop from
```
Comment (schveiguy) [inline std/range/package.d]: `range`, not `r`.

**Comment by JackStouffer** [comment]
@schveiguy fixed

**Comment by schveiguy** [comment]
Auto-merge toggled on

**Pull request merged**